src/pkg/services/m365/api/channels.go

```diff
@@ -169,15 +174,19 @@ func ChannelMessageInfo(
 		msgCreator = ptr.Val(from.GetUser().GetDisplayName())
 	}
 
+	if msg.GetBody() != nil {
+		content = ptr.Val(msg.GetBody().GetContent())
+	}
```

Comment:
Are we handling the replies of type other than `text` message. For eg, an attachment reply?

Reply:
All replies are ChatMessageable types.  An attachment-only reply may have zero length content, but it will otherwise function as normal.